Teensy/src/gpio/gpio.cpp

```diff
@@ -11,59 +11,59 @@
 #include <Arduino.h>
 #include "gpio.h"
 
-static int32_t gpio_getWheelSpeed();
-static uint32_t gpio_getEngineRevs();
-static int32_t gpio_getDamperPosition();
-static int32_t gpio_getGearPosition();
-static uint32_t gpio_getSteeringWheelPosition();
-static int32_t gpio_getStrain();
+static daq_WheelSpeed_t gpio_getWheelSpeed();
+static daq_EngineRev_t gpio_getEngineRevs();
+static daq_DamperPos_t gpio_getDamperPosition();
+static daq_GearPos_t gpio_getGearPosition();
+static daq_SteeringWhlPos_t gpio_getSteeringWheelPosition();
+static daq_Strain_t gpio_getStrain();
```

Comment:
Needs it's own data type to store 20 strain measurements, however, not essential right now as I doubt we'll be able to gather any strain data this year. We'll probably need to make a seperate stripped down version of the DAQ system just to gather strain for one test day down the line.

Reply:
Ok cool, I've decided just to remove it in that case :)